stix_shifter_modules/crowdstrike_logscale/README.md

```diff
@@ -586,15 +621,15 @@ crowdstrike_logscale
                 },
                 "4": {
                     "type": "process",
-                    "command_line": "\"C:\\Windows\\system32\\cmd.exe\" ",
+                    "command_line": "\"C:\\Windows\\System32\\cmd.exe\"  /c \"\"C:\\Program Files\\Amazon\\EC2Launch\\EC2Launch.exe\" wallpaper --path=\"C:\\ProgramData\\Amazon\\EC2Launch\\wallpaper\\Ec2Wallpaper.jpg\" --attributes=\"hostName,instanceId,privateIpAddress,publicIpAddress,instanceSize,availabilityZone,architecture,memory,network\" \"",
                     "binary_ref": "5",
-                    "x_process_graph_id": "pid:7xyz:184941492793"
+                    "x_process_graph_id": "pid:7adb:232200947972"
                 },
                 "5": {
                     "type": "file",
                     "hashes": {
-                        "MD5": "e7a6babc90f4",
-                        "SHA-256": "eb71xxxxx08"
+                        "MD5": "e7a6b1f51efb405287a8048cfa4690f4",
```

Comment:
This should probably be obscured (if it's not already)

Reply:
modified as suggested